content/docs/02.5-architecture/04.1-grps-proxy.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "gRPC and Proxies"
+metaTitle: "gRPC and Proxies"
+metaDescription: "Learn about gRPC and how a proxy is used to communicate between the management platform and the workload cluster."
+icon: ""
+hideToC: false
+fullWidth: false
+---
+ 
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# gRPC and Proxies
+
+Palette uses [gRPC](https://grpc.io) to communicate between the management platform and the workload cluster. gRPC is a high-performance, open-source universal RPC framework. It is used to build distributed applications and services. gRPC is based on HTTP/2 and uses protocol buffers ([protobuf](https://protobuf.dev/)) as the underlying data serialization framework. 
+
+<br />
+
+<InfoBox>
+
+Refer to the [Network Ports](/architecture/networking-ports) documentation for a detailed network architecture diagram with gRPC and to learn more about the ports used for communication.
+
+</InfoBox>
+
+<br />
+
+When gRPC is used with network proxies, the proxy servers may or may not support gRPC or require additional configuration to allow gRPC traffic to pass through. The following table summarizes the different scenarios and whether or not the proxy server supports gRPC.
+
+
+| **Scenario** | **Description** | **Proxy Supported** |
+|:-------------|:----------------|:--------------------|
+| gRCP with HTTP/HTTPS - No SSL bump| gRPC traffic is sent over HTTP/HTTPS ,and the proxy does not perform Secure Socket Layer (SSL) bump. This is universally supported. | ✅ |
+| gRPC with HTTP/HTTPS - SSL bump | gRPC traffic is sent over HTTP/HTTPS ,and the proxy performs SSL bump. Support vary by vendor. | ⚠️ |
+| gRPC with [Squid](https://wiki.squid-cache.org) Open Source Proxy | gRPC traffic is sent over HTTP/HTTPS ,and the proxy performs SSL bump. Supported in some scenarios but require additional configuration. | ❌ or ⚠️ |
+
+
+The following sections provide more information about gRPC and proxies.
+
+<br />
+
+ ## Proxy Without SSL Bump
+
+Because gRPC is based on HTTP/2, any proxy server that supports the [HTTP CONNECT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/CONNECT) method can be used to forward gRPC traffic. No configuration is required for this scenario. The exception is when the proxy server performs an SSL bump, discussed in the next section.
```

Comment:
Let's add a link to the section instead of "discussed in the next section". We don't want to assume sequential reading. Also, another section or subsection could potentially be added between these two sections if another writer isn't watching for something like this.